lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -61,14 +61,71 @@
     default: "present"
     choices: ["present", "absent", "enabled", "disabled"]
   telephonenumber:
-    description:
-    - List of telephone numbers assigned to the user.
-    - If an empty list is passed all assigned telephone numbers will be deleted.
-    - If None is passed telephone numbers will not be checked or changed.
+    description: List of telephone numbers assigned to the user.
+    required: false
+  mobile:
+    description: List of mobile telephone numbers assigned to the user.
+    required: false
+  pager:
+    description: List of pager numbers assigned to the user.
+    required: false
+  facsimiletelephonenumber:
+    description: List of fax numbers assigned to the user.
     required: false
   title:
-    description: Title
+    description: Job Title
+    required: false
+  initials:
+    description: initials
+    required: false
+  random:
+    description:
+    - Generate a random password if the user is created.
+    - If the user is modified this parameter is ignored.
+    - To enforce a new random password for a user set the new_random flag.
+    required: false
+    default: false
+  new_random:
+    description:
+    - This parameter is ignored if the user is created.
+    - If the user is modified and this flag is true, a new random password is generated each time Ansible runs.
+    - Normally you don't want to set this true, since this resets the password when Ansible runs.
+    required: false
+    default: false
+  street:
+    description: Street address
+    required: false
+  l:
+    description: City
+    required: false
+  st:
+    description: State/Province
+    required: false
+  postalcode:
+    description: ZIP
+    required: false
+  ou:
+    description: Org. Unit
+    required: false
+  manager:
+    description: Manager
     required: false
+  carlicense:
+    description: Car License
+    required: false
+  homedirectory:
+    description: Home directory
+    required: false
+  uidnumber:
```

Comment:
Would be nice to have an alias for the parameter to be backwards compatible with the original implementation which used _uid_.